docs/results.rst

```diff
@@ -108,6 +108,12 @@ configurations:
    Must have access to specified folder.
 -  mime_type: Media type of the file to be uploaded. If none is specified
    it will take the type of the filename extension.
+-  start_timeout, max_timeout, retry_status_codes: drive API calls sometimes
+   fail with 500 errors. To work around this behaviour, in case of error the
+   call is retried after waiting *start_timeout* (2 by default) seconds,
+   doubling the waiting time after each error until reaching *max_timeout* (300
```

Comment:
I would be nice to mention what happens when it reaches `max_timeout`. An exception or what?

Reply:
Done 👍